siemens_data.py

```diff
@@ -0,0 +1,74 @@
+import csv
+import sys
+import pandas as pd
+from datetime import datetime
+
+""" Class for reading and manipulating Siemens data from transformed CSVs. 
+	Contains API for accessing data, as well as function for reading the CSV's. 
+"""
+
+class SiemensData:
+
+	data = None
+	datetimes = []
+
+	# Open and store transformed CSV data as class pd object. 
+	def read_csv(self, filename): 
+		self.data = pd.read_csv(filename, dtype=object)
+
+	def find_data_for_room(self, room_name):
+		equipment_dictionary = {} #Key = equipment name, value = sorted list of tuples containing (timestamp, data)
+
+		if self.data is None:
+			raise Exception("You haven't read in a csv yet!")
+
+		for header in self.data:
+			if room_name in header:
+				equipment_data = [] # Initialize container for (timestamp, data) tuples
+
+				equipment_name = self.find_equipment_name(header) #Key for dictionary
+				if len(self.datetimes) < 1: 
+					raise Exception("You haven't parsed the timestamps yet!")
+
+				for i in range(len(self.datetimes)):
+					datetime = self.datetimes[i]
+					data = self.data.get_value(i, header)
+					equipment_data.append((datetime, data))
+
+				equipment_dictionary[equipment_name] =  equipment_data
+
+		return equipment_dictionary
+
+
+
+	def find_equipment_name(self, header):
+		parts_of_header = header.split(".")
+		return parts_of_header[-1]
+
+	def create_datetimes_list(self):
+		datetimes = []
+		for i in range(len(self.data.columns.values)): # Iterate through all rows
+			# Get date and time from particular column
+			date = self.data.get_value(i, "<>Date")
+			time = self.data.get_value(i, "Time")
+
+			timestamp = date + " " + time
+			datetime_object = datetime.strptime(timestamp, "%m/%d/%y %H:%M:%S") #create datetime object from MM/DD/YY xx:xx:xx format
+			datetimes.append(datetime_object)
+
+		self.datetimes = datetimes
+
+
+
+def main():
+	filename = "betterTestCSV.csv"
+	data_reader = SiemensData()
+	data_reader.read_csv(filename)
+	data_reader.create_datetimes_list()
+	equipment_dictionary = data_reader.find_data_for_room("ACDIN.AH1")
+	print(equipment_dictionary)
+
+
+
+if __name__ == '__main__':
+	main()
```

Comment:
I don't know if this matters to us, but one of my coworkers over the summer was very particular that files should end in new lines.  I don't know if it matters at all, but I did look it up and found [this](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) semi-interesting answer